string                              id
bool                                assigned
float                              deadline
string                              tasks
mapc_rhbp_ettlinger/AssembleBid     bid
